softlayer/vm/network.go

```diff
@@ -24,3 +24,8 @@ func (ns Networks) First() Network {
 }
 
 func (n Network) IsDynamic() bool { return n.Type == "dynamic" }
+
+func (n Network) AppendDNS(addr string) Network {
```

Comment:
Where is the test for this method? Is it still needed? If not remove it, if yes, then add a test.